src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -100,7 +100,11 @@
             fxFlex
             fxLayout
             fxLayoutAlign="end"
-            *ngIf="!orgObj?.sourceControl.startsWith('dockstore.org') && (entryType$ | async) !== EntryType.Tool"
+            *ngIf="
+              !orgObj?.sourceControl.startsWith('dockstore.org') &&
+              (entryType$ | async) !== EntryType.Tool &&
+              (entryType$ | async) !== EntryType.Notebook
```

Comment:
You could add a check for `EntryType.Service` here and around line 83, and that would fix dockstore/dockstore#3777 while you're here, but you could also argue it's beyond the scope of the issue and riskier. I'm good either way, and there's a ticket to capture services already if you decide not to.

Stumbled across this as I was reviewing and wondering why services didn't have this problem as well, and why you didn't need to add a check for it.